### ISSUE TITLE
[Suggestion] Add Wolf3D's Main menu, options and save features

### ISSUE THREAD

**Issue by ShadowOne333** [libretro/ecwolf#65]
This is probably already in the to-do listed, but I wanted to create an issue report to keep track of it.
Basically, re-implement the original Wolfenstein 3D main menu, its options menu and the save/loading slots feature as well.

![Wolf3D](http://www.vgmpf.com/Wiki/images/thumb/b/b6/Wolfenstein_3D_-_DOS_-_2.png/256px-Wolfenstein_3D_-_DOS_-_2.png)

I am not sure if this was added into the original ECWolf and is pending on the libretro port or not, but I wanted to leave this for tracking.

**Comment by phcoder** [comment]
This is controlled through libretro menuss

**Comment by ShadowOne333** [comment]
It'd still be nice to have access to the original main menu of Wolf3D (alongside the whole bottom section of options), as it's an iconic part of the game itself alongside the menu song.

**Comment by phcoder** [comment]
> It'd still be nice to have access to the original main menu of Wolf3D (alongside the whole bottom section of options), as it's an iconic part of the game itself alongside the menu song.

Well it's all open-source. Feel free to provide a patch

**Comment by LibretroAdmin** [comment]
Should be added now
